From 20135166521b601da1d4a9dc6c13b58fd3847735 Mon Sep 17 00:00:00 2001
From: David Edmundson <kde@davidedmundson.co.uk>
Date: Wed, 27 Jan 2021 10:56:43 +0000
Subject: [PATCH 14/21] Fix the systemd wayland boot

Plasma-workspace currently starts kwin_wayland before afterwards
continuing to spawn the full session, through whatever mechanism.

We ultimately want to just have systemd manage everything all at once,
but this was not realised in time for 5.21 due to a problem of
propogating environment variables.

By removing this file we go to a working state with the option enabled,
and can build on it for the next release.

BUG: 432189


(cherry picked from commit ca81be6635131bfe2d290022128d0569357d2dd5)
---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0e00408e8..f10361b62 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -814,5 +814,8 @@ configure_package_config_file(KWinDBusInterfaceConfig.cmake.in
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/KWinDBusInterfaceConfig.cmake
     DESTINATION ${CMAKECONFIG_INSTALL_DIR})
 
-ecm_install_configured_files(INPUT plasma-kwin_x11.service.in plasma-kwin_wayland.service.in @ONLY
+
+# plasma-kwin_wayland.service.in is currently disabled as plasma-workspace currently launches
+# kwin-wayland explicitly. Porting this is currently blocked on propogating DISPLAY
+ecm_install_configured_files(INPUT plasma-kwin_x11.service.in @ONLY
             DESTINATION ${SYSTEMD_USER_UNIT_INSTALL_DIR})
-- 
2.29.2

